frontend/components/queries/QueryProgressDetails/QueryProgressDetails.jsx

```diff
@@ -66,9 +66,15 @@ const QueryProgressDetails = ({ campaign, className, onRunQuery, onStopQuery, qu
     <div className={`${baseClass} ${className}`}>
       <div className={`${baseClass}__wrapper`}>
         <div className={`${baseClass}__text-wrapper`}>
-          <span className={`${baseClass}__text-online`}>Online - {onlineHostsTotalDisplay} returning {onlineResultsTotalDisplay}</span>
-          <span className={`${baseClass}__text-offline`}>Offline - {offlineHostsTotalDisplay} returning 0 results</span>
-          <span className={`${baseClass}__text-error`}>Failed - {failedHostsTotalDisplay} returning {totalErrorsDisplay}</span>
+          <span className={`${baseClass}__text-online`}>{hostsCount.successful} out of {onlineHostsTotalDisplay} responding
+            <span className={`${baseClass}__text-results`}> and returning {onlineResultsTotalDisplay}</span>
```

Comment:
Maybe a tad simpler to read?

```suggestion
            <span className={`${baseClass}__text-results`}> returning {onlineResultsTotalDisplay}</span>
```